### ISSUE TITLE
Name field in tree-related datasets being [blank] is flagging issues

### ISSUE THREAD

**Issue by MJFaucherFolie** [digital-land/submit#548]
## Description
We have received a message from an St Albans reviewing their TPO dataset on the dashboard. It has flagged that the NAME field being BLANK is an issue, but this is acceptable practice according to the [guidance](https://www.planning.data.gov.uk/guidance/specifications/tree-preservation-order). 

## Additional context:
![Image](https://github.com/user-attachments/assets/930f77cb-1d41-4338-a8eb-107816e61fb3)

[St Albans council](https://submit.planning.data.gov.uk/organisations/local-authority:SAL/tree-preservation-order) have a task to fill in empty NAME fields. This does not comply with the guidance. 

![Image](https://github.com/user-attachments/assets/cf28b5cb-fa81-4f76-a557-160b9a16de53)
Users who check their data using the check-tool will not be able to progress and submit their data if the name field is blank. [This is an instance of the check-tool displaying the issue](https://submit.planning.data.gov.uk/check/results/TbdZJxaJR7eMPtQ554Nhwv/0)

**Comment by MJFaucherFolie** [comment]
Issue has been addressed by the Infrastructure team.

**Comment by MJFaucherFolie** [comment]
Cannot assess if St Albans is still experiencing this issue as their endpoints are 404'ing

**Comment by MJFaucherFolie** [comment]
Reached out to members of the Infrastructure team to confirm if it's something they should fix or something we need to address

**Comment by MJFaucherFolie** [comment]
Had a chat with Ben in Infra to understand this bug further.

TL;DR - This will add to the tech debt impacting the pipeline.

The ideal model that Infra want to move to is the Pipeline reviewing specifications on a daily basis and surfacing issues based on those specifications. Changes to the specifications would result in changes to the types of issues displayed.

To address this issue, adjustments will need to be made directly to the pipeline and the configuration files, which will increase tech debt.

This is likely to be addressed in the next quarter

**Comment by CharliePatterson** [comment]
Thanks @MJFaucherFolie - could you please make sure that the ticket they have on their backlog has all the relevant information from this one, and then this can be closed 🙏

**Comment by Ben-Hodgkiss** [comment]
@MJFaucherFolie @CharliePatterson - I've added this ticket to our backlog (see right-hand panel). Rather than closing this, if you just remove it from your backlog, I think it'll still continue to exist in ours!

**Comment by CharliePatterson** [comment]
@Ben-Hodgkiss @eveleighoj if we removed this field from the mandatory fields list, would it resolve this issue?
